features/specs/gezag-minderjarige/3.2-indicatie-gezag-minderjarige.feature

```diff
@@ -152,17 +147,13 @@ Functionaliteit: 3.2 - Achterhalen gezag na uitspraak
     Dan heeft de response een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 000000091 |
-#      En heeft de persoon een 'gezag' met de volgende gegevens
-#      | naam                             | waarde                    |
-#      | type                             | GezamenlijkGezag          |
-#      | minderjarige.burgerservicenummer | 000000091                 |
-#      | ouder.burgerservicenummer        | 000000061                 |
-#      | derde.burgerservicenummer        | 000000085                 |
     En heeft de persoon een 'gezag' met de volgende gegevens
-      | naam                             | waarde    |
-      | type                             | Voogdij   |
-      | minderjarige.burgerservicenummer | 000000091 |
-    En heeft 'gezag' geen derden
+      | naam                             | waarde           |
+      | type                             | GezamenlijkGezag |
+      | minderjarige.burgerservicenummer | 000000091        |
+      | ouder.burgerservicenummer        | 000000061        |
+      | derde.burgerservicenummer        | 000000085        |
```

Comment:
er is uitspraak 1D, dus verwacht ik een onbekende derde.

de titel van het scenario die zegt dat de "vader" - die volgens de gegeven stappen juridisch geen vader is - het gezag heeft. Deze "vader" zou de gezaghebbende kunnen zijn, maar dat hoeft niet.

ik denk dat het verwarrend om te spreken over "vader" wanneer deze persoon juridisch geen ouder is.

het voorbeeld is ook raar. als "Iefke" een vrouw is (moeder) en "Paul" een man, dan zou Paul ook juridisch de vader moeten zijn, omdat het kind uit hun huwelijk is geboren. Had de partner "Paula" geheten (vrouw) dan had dit wel gekund. Dit verandert dan niks aan de uitkomst want die blijft gebaseerd op de gerechtelijke uitspraak 1D

Reply:
Je hebt gelijk. Er is verwarring ontstaan wat wij verstaan onder _niet ouder_ (derde) vanuit het domein en _niet ouder_ vanuit gezag-api.

Gezag uitspraak bevat nu `OnbekendeDerde`.